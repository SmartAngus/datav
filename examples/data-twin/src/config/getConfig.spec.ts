import { getConfig } from './getConfig'

import {
  FLAVOR_PRODUCTION,
  FLAVOR_STAGING,
  FLAVOR_DEVELOPMENT,
} from './constants'

test.each([FLAVOR_PRODUCTION, FLAVOR_DEVELOPMENT, FLAVOR_STAGING] as const)(
  '%s matches snapshot',
  (flavor) => {
    expect(getConfig(flavor, true)).toMatchSnapshot()
  }
)
