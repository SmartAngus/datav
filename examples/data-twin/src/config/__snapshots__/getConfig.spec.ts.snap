// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`development matches snapshot 1`] = `
Object {
  "EXAMPLE": "string-in-development",
  "SENTRY_DSN": "",
  "SENTRY_ENABLED": "0",
}
`;

exports[`production matches snapshot 1`] = `
Object {
  "EXAMPLE": "string-in-production",
  "SENTRY_DSN": "",
  "SENTRY_ENABLED": "1",
}
`;

exports[`staging matches snapshot 1`] = `
Object {
  "EXAMPLE": "string-in-staging",
  "SENTRY_DSN": "",
  "SENTRY_ENABLED": "1",
}
`;
